app/controllers/admin/taggings_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Admin
+  class TaggingsController < ApplicationController
+    before_action(:authenticate_user!) if defined?(Devise)
+    before_action { EffectiveResources.authorize!(self, :admin, :effective_pages) }
+
+    include Effective::CrudController
+
+    page_title 'Tagged content'
```

Comment:
We can remove these page_titles, I think that they are set automatically based on the pluralized model name